packages/pangea-sdk/pangea/config.py

```diff
@@ -2,26 +2,16 @@
 # Author: Pangea Cyber Corporation
 
 from dataclasses import dataclass
-from typing import Literal, Optional
+from typing import Optional
 
 
 @dataclass
 class PangeaConfig:
     """Holds run time configuration information used by SDK components."""
 
-    domain: str = "aws.us.pangea.cloud"
+    base_url_template: str = "https://{SERVICE_NAME}.aws.us.pangea.cloud/"
```

Comment:
It feels unnecessarily risky to me to have this be a breaking change, couldn't Config object allow for either domain or  base_url_template keyword parameter, and then this can be added instead of replacing?